app/client/components/modal/ModalFrame.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const ModalFrame = (props) => {
```

Comment:
Wow, so you do know how to write stateless React component pure functions using arrow functions